### PR TITLE
DSD-2602 created helm chart for bqat service

### PR DESCRIPTION
**Pull request by bhumi46**

### CODE REVIEW

File: charts/bqat-sdk/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+name: bqat-sdk
+description: A Helm chart for MOSIP Auditmanager module
+type: application
+version: 12.0.1-B3
```
Comment (ckm007) [inline charts/bqat-sdk/Chart.yaml:5]: since this is first version. Lets make the chart version to 1.0.0.

File: charts/bqat-sdk/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+name: bqat-sdk
+description: A Helm chart for MOSIP Auditmanager module
```
Comment (syedsalman3753) [inline charts/bqat-sdk/Chart.yaml:3]: please update description

File: charts/bqat-sdk/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  - name: common
+    repository: https://charts.bitnami.com/bitnami
+    tags:
+      - bitnami-common
+    version: 1.x.x
+home: https://mosip.io
+keywords:
+  - mosip
+  - bqat-sdk
+  - kernel
```
Comment (syedsalman3753) [inline charts/bqat-sdk/Chart.yaml:17]: please remove kernel from here as kernel has no link to sdk

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]
can you confirm if the deployment has been tested?

**Review by syedsalman3753 — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Review by syedsalman3753 — APPROVED** [review]

**Review by syedsalman3753 — APPROVED** [review]

**Pull request merged**